### PR TITLE
[tempo] Enabled the opencensus receive in the default Helm template for Tempo

### PR DESCRIPTION
**Pull request by rsanting**
In the Tempo values file it is mentioned that all ports and protocols that Tempo supports are enabled. However not all of them actually are enabled. I added opencensus, there are more like Zipkin that I could add but I haven't tested them personally. I did test Opencensus and that works.

### CODE REVIEW

**Comment by joe-elliott** [comment]
Thanks for the addition! Will merge once PR passes.

**Comment by joe-elliott** [comment]
You may need to push an empty commit to get CI to wake up. I can't seem to force it to retry through the github UI.

Also, you will need to increment the chart version and re-run helm-docs so you may as well do that in the commit :).

**Comment by rsanting** [comment]
Thanks, I have to admit I forgot about the chart version and I didn't know about the helm-docs. Next time I will come better prepared. All should be well in the latest commit. Let's hope CI runs.

**Comment by rsanting** [comment]
Ci ran and it came up with two errors. I fixed them in a force push but if that stops CI from running again I can push an empty commit.

**Comment by rsanting** [comment]
Sorry for the mess, I'm not used to running helm-docs manually and I messed up the tag one time. Now all should be fine. Looking at the helm-docs though, would a pull request that adds pre-commit configs be useful or is that not wanted?

**Comment by joe-elliott** [comment]
> Sorry for the mess, I'm not used to running helm-docs manually and I messed up the tag one time.
Not at all! Thank you for the contribution.

> Looking at the helm-docs though, would a pull request that adds pre-commit configs be useful or is that not wanted?
I dunno. Unsure what you mean by pre-commit configs. I'm not super familiar with helm-docs or the way this CI is setup. I just know it makes us run it when we submit a PR.  Generally I'm open to any improvement.

Thanks again!

File: charts/tempo/Chart.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: tempo
 description: Grafana Tempo Single Binary Mode
 type: application
-version: 0.7.4
-appVersion: v1.0.1
+version: 0.7.5
+appVersion: v1.0.2
```
Comment (joe-elliott) [inline charts/tempo/Chart.yaml:6]: let's keep this at 1.0.1. this should match the actual tempo version which will remain 1.0.1.

**Review by joe-elliott — CHANGES_REQUESTED** [review]

**Comment by rsanting** [comment]
Pre-commit is a framework for managing and maintaining multi-language pre-commit hooks. If we add a config for it anyone that has pre-commit installed and enables it wouldn't need to run the helm-docs by hand. I will test it out with this repo and just make a pull-request, it could always be denied.

**Review by joe-elliott — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**